neptunecontrib/viz/__init__.py

```diff
@@ -13,3 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+from neptunecontrib.viz.experiments import *
```

Comment:
Please import only what we actually want to expose. Alternatively use `__all__`.

Reply:
Done